core/backend/src/IModelDb.ts

```diff
@@ -305,6 +305,13 @@ export abstract class IModelDb extends IModel {
     GeoCoordConfig.loadForImodel(this.workspace.settings); // load gcs data specified by iModel's settings dictionaries, must be done before calling initializeIModelDb
 
     this.initializeIModelDb();
+    this.onChangesetApplied.addListener(() => {
+      const extents = this.queryFilePropertyString({ name: "Extents", namespace: "dgn_Db" });
+      // TODO: What if extents is undefined? And was previously defined? Is this a possibility?
```

Comment:
Any thoughts on this? My guess is that it is unlikely for an imodel to go from having project extents to no project extents so it is not worth handling, but not sure.

Reply:
I dont think its possible to from defined extents to undefined, the other way around yes